src/main/java/net/sf/jabref/gui/errorconsole/ErrorConsoleView.java

```diff
@@ -90,38 +99,32 @@ private void closeErrorDialog() {
     private void listViewStyle() {
         // Handler for listCell appearance (example for exception Cell)
         allMessages.setCellFactory(
-                new Callback<ListView<LogMessage>, ListCell<LogMessage>>() {
+                new Callback<ListView<LogEvent>, ListCell<LogEvent>>() {
                     @Override
-                    public ListCell<LogMessage> call(
-                            ListView<LogMessage> listView) {
-                        return new ListCell<LogMessage>() {
+                    public ListCell<LogEvent> call(
+                            ListView<LogEvent> listView) {
+                        return new ListCell<LogEvent>() {
 
                             @Override
-                            public void updateItem(LogMessage logMessage, boolean empty) {
+                            public void updateItem(LogEvent logMessage, boolean empty) {
                                 super.updateItem(logMessage, empty);
                                 if (logMessage != null) {
-                                    setText(logMessage.getMessage());
-
-                                    MessageType prio = logMessage.getPriority();
-                                    switch (prio) {
-                                        case EXCEPTION:
-                                            getStyleClass().add("exception");
-                                            setGraphic(IconTheme.JabRefIcon.INTEGRITY_FAIL.getGraphicNode());
-                                            break;
-                                        case OUTPUT:
-                                            getStyleClass().add("output");
-                                            setGraphic(IconTheme.JabRefIcon.INTEGRITY_WARN.getGraphicNode());
-                                            break;
-                                        case LOG:
-                                            getStyleClass().add("log");
-                                            setGraphic(IconTheme.JabRefIcon.INTEGRITY_INFO.getGraphicNode());
-                                            break;
-                                        default:
-                                            break;
+                                    setText(logMessage.getMessage().toString());
+
+                                    Level prio = logMessage.getLevel();
+                                    if (prio.equals(ERROR)) {
```

Comment:
pls use a switch statement here and show an info by default (instead of nothing with `null/null`)